tests/test_launch_methods/test_jsrun/jobs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Begin LSF Directives
+#BSUB -P BIP178
+#BSUB -W 0:30
+#BSUB -nnodes 2
+#BSUB -J jsrun_test
+#BSUB -o jsrun_test.%J
+#BSUB -e jsrun_test.%J
+#BSUB -N rct-tests@googlegroups.com
+
+# CONFIG
+path=`pwd`
+d=`date +%D | sed 's/\//_/g'`
+host=`/bin/hostname`
+#dest="rct-tests@googlegroups.com"
+dest="vivek.bala@rutgers.edu"
+
+run_cmd() {
+    res_set=$1
+    omp_env=$2
+    cmd=$3
+    exp_out=$4
+    tid=$5
+
+    # Create required folders
+    rm results_$d/$tid -rf
+    mkdir results_$d/$tid
+
+    # Record resource file, omp env, and cmd in log
```

Comment:
What omp stands for?

Reply:
It is for OPENMP

Reply:
Yes it stands for OpenMP. The value is being extracted from the scripts and executed in the current session.